bibblio.py

```diff
@@ -228,14 +357,27 @@ def items_that_need_coverage(self, identifiers=None, **kwargs):
                 .outerjoin(CustomListEntry.customlist)\
                 .outerjoin(edition_entry, Edition.custom_list_entries)\
                 .outerjoin(edition_list, edition_entry.customlist)\
-                .filter(or_(
-                    CustomList.id==self.custom_list.id,
-                    edition_list.id==self.custom_list.id))\
+                .join(Work.license_pools).join(LicensePool.identifier)\
+                .filter(
+                    or_(
+                        CustomList.id==self.custom_list.id,
+                        edition_list.id==self.custom_list.id
+                    ),
+                    not_(
+                        or_(
+                            LicensePool.suppressed==True,
+                            LicensePool.superceded==True
+                        )
+                    ),
```

Comment:
Not a big deal but this would be a little simpler if you filtered on suppressed==False and superceded==False. You might also try using `Library.restrict_to_ready_deliverable_works`, which is newly available in core.